_episodes/interviews/2023-03-03-joshuan-nelson.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Rustdoc with Joshua Nelson"
+date: 2023-03-03T16:47:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-03-03-joshua-nelson.mp3
+duration: "48:52"
+length: "46918426"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Joshua Nelson](https://jyn.dev/), team lead for the [Bootstrap](https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html) team for Rust, and formally on the Rustdoc team for Rust.
```

Comment:
I couldn't find a link for the bootstrap team. If we find one, we should include it here.